mixer/pkg/attribute/envoyProtoBag.go

```diff
@@ -51,9 +51,10 @@ var envoyProtoBags = sync.Pool{
 }
 
 func fillAddress(reqMap map[string]interface{}, address *core.Address, name string) {
-	socketaddress := address.GetSocketAddress()
-	reqMap[name+".ip"] = []byte(net.ParseIP(socketaddress.GetAddress()).To16())
-	reqMap[name+".port"] = int64(socketaddress.GetPortValue())
+	if socketaddress := address.GetSocketAddress(); socketaddress != nil {
```

Comment:
Should we add
```
if address == nil {
  return
}
```
before the first line

Reply:
I don't know if there's ever a socket address without the address field that will cause that line to be hit but I will add it to be safe.